packages/fast-development-site-react/src/components/site/dev-tools.tsx

```diff
@@ -10,7 +10,7 @@ export enum FrameworkEnum {
     angular = "angular"
 }
 
-export type tabType = "code" | "properties" | "schema";
+export type tabType = "Code" | "Properties" | "Schema";
```

Comment:
@janechu should we consider using an enum here?

Reply:
I think this is fine for now, it's type safe

Reply:
New task created by @janechu for this. Not a pri and we currently have type safety, so put into the backlog as a good first issue for contributors: https://github.com/Microsoft/fast-dna/issues/555